docs/release-notes.txt

```diff
@@ -8,6 +8,53 @@
  Version 0.13
 **************
 
+Version 0.13.4
+==============
+
+**Release Date:** September 16, 2020
+
+**Improvements**
+
+- Warn users when setting ``DistributeConfig`` on an Estimator. Configuring an Estimator via ``tf.distribute.Strategy`` can conflict with how Determined performs distributed training. With this change, Determined will attempt to catch this problem and surface an error message in the experiment logs. An Estimator can still be configured with an empty ``DistributeConfig`` without issue.
+
+- Support configuring default task images, Docker pull policy, and Docker registry credentials via the :ref:`master-configuration` and the :ref:`helm-config`. Note that individual task configurations will continue to overwrite these defaults.
+
+- Add connection checks for dynamic agents. A dynamically provisioned agent will be terminated if it is not actively connected to the master for at least five minutes.
+
+- Remove support for ``dataflow_to_tf_dataset`` in EstimatorTrial. Dataflows should be wrapped using ``wrap_dataset(shard=false)`` instead.
```

Comment:
EstimatorTrial should be double-backticked.

Reply:
fixed